dynamic-config.md

```diff
@@ -24,7 +24,7 @@ This section describes the common operations of dynamic configuration change.
 
 To view the configuration of all instances in the cluster, use the `show config` SQL statement. The result is as follows:
 
-```
+```sql
 show config;
```

Comment:
Separate this code from the execution result.

Reply:
Done. PTAL.